script/magic_numbers/decimal_multiplication_generator.py

```diff
@@ -0,0 +1,47 @@
+import math
+
+from random import seed
+from random import randint
+# seed random number generator
```

Comment:
could you similarly please run intellij/clion/something autoformat on this file? thanks!

Reply:
done